Orange/ensembles/ada_boost.py

```diff
@@ -32,11 +44,22 @@ class SklAdaBoostRegressor(SklModelRegression):
 class SklAdaBoostRegressionLearner(SklLearnerRegression):
     __wraps__ = skl_ensemble.AdaBoostRegressor
     __returns__ = SklAdaBoostRegressor
-    name = 'skl adaBoost regression'
 
     def __init__(self, base_estimator=None, n_estimators=50, learning_rate=1.,
                  loss='linear', random_state=None, preprocessors=None):
+        # If fitter, get the appropriate Learner instance
+        if isinstance(base_estimator, Fitter):
+            base_estimator = base_estimator.get_learner(
+                base_estimator.REGRESSION)
+        # If sklearn learner, get the underlying sklearn representation
         if isinstance(base_estimator, SklLearner):
             base_estimator = base_estimator.__wraps__(**base_estimator.params)
         super().__init__(preprocessors=preprocessors)
         self.params = vars()
+
+
+class SklAdaBoostLearner(Fitter):
```

Comment:
Fitters for KNN and Tree went into _Orange/modelling_. They might as well (or even rather) be defined closer to where their underlying learners are, but let's just be consistent.